### ISSUE TITLE
python-imaging not available on Ubuntu 18.04 LTS

### ISSUE THREAD

**Issue by Jnesselr** [kimchi-project/kimchi#1227]
I'm trying to install kimchi, but apt is throwing this error:
```
Package python-imaging is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
However the following packages replace it:
  python-pil

E: Package 'python-imaging' has no installation candidate
```

Does kimchi only work with python 2? python-imaging is only available for python 2 I believe and Ubuntu 18.04 is python3 by default.

**Comment by PMaynard** [comment]
I got it working by installing ```python-pil```, and ignoring the need for python-imaging. Since pil is a fork of python-imaging.

**Comment by Erlenbush** [comment]
I tried that, but with python-pil kimchi still doesn't seem to want to start. I have wok up and running but can't seem to get Kimchi going.

**Comment by PMaynard** [comment]
What errors are you seeing, post them here if you can.

**Comment by djtobyy** [comment]
Got the same Problem:
After Installing all there are dependencies what can't be solved: (Sorry my OS is GERMAN :) 👎 )


PS: It says: "kimchi is depending on PACKAGENAME but package PACKAGENAME is not installed."
` Vorbereitung zum Entpacken von kimchi-2.5.0-0.noarch.deb ...
Entpacken von kimchi (2.5.0) ...
dpkg: Abhängigkeitsprobleme verhindern Konfiguration von kimchi:
 kimchi hängt ab von python-imaging; aber:
  Paket python-imaging ist nicht installiert.


 kimchi hängt ab von novnc; aber:
  Paket novnc ist nicht installiert.


 kimchi hängt ab von python-libvirt; aber:
  Paket python-libvirt ist nicht installiert.


 kimchi hängt ab von python-ethtool; aber:
  Paket python-ethtool ist nicht installiert.


 kimchi hängt ab von python-ipaddr; aber:
  Paket python-ipaddr ist nicht installiert.


 kimchi hängt ab von python-guestfs; aber:
  Paket python-guestfs ist nicht installiert.


 kimchi hängt ab von python-magic; aber:
  Paket python-magic ist nicht installiert.


 kimchi hängt ab von python-paramiko; aber:
  Paket python-paramiko ist nicht installiert.`

**Comment by rickyphewitt** [comment]
I ran into this same issue. I was able to install python-pil instead of python-imaging. Then run the command:
`dpkg --ignore-depends=python-imaging -i your-kimchi-deb-here `
Then opened up the wok web interface and enabled the plugin.

**Comment by Erlenbush** [comment]
sudo apt install python-paramiko fixed it:)

**Comment by tekgeek1205** [comment]
So with the help of this thread, I've got everything up and running except I'm still getting this error when I try to install anything else.

kimchi : Depends: python-imaging but it is not installable

If I run apt --fix-broken install It uninstall kimchi. Then I can install what I need. What a pain. How do I make apt ignore this missing dependency when Installing other stuff. Python-pil seem to be working.

**Comment by juan-vg** [comment]
@tekgeek1205 you can edit dependencies by hand

https://serverfault.com/a/383958/471258

**Comment by tekgeek1205** [comment]
thanks for responding, I figured it out at like 2 in the morning after chasing a different rabbit down it's hole. Then forgot to update my post the next morning.

**Comment by damaspi** [comment]
I did the following to remove the python-imaging dependency:

- download the kimchi package `wget https://github.com/kimchi-project/kimchi/releases/download/2.5.0/kimchi-2.5.0-0.noarch.deb` 
- Extract the contents `ar -x kimchi-2.5.0-0.noarch.deb`
- Unpack control archive `tar xzf control.tar.gz`
- Edit the control and control.in files and replace python-imaging with python-pil
- Repack control.tar.gz: `tar c Makefile Makefile.am Makefile.in control.in control | gzip -c > control.tar.gz` 
- Repack .deb: `ar rcs kimchi-2.5.0-0-pil.noarch.deb debian-binary control.tar.gz data.tar.xz`

**Comment by alinefm** [comment]
Fixed by bc66961b5c1ca1d257df4ec5caf46f163512e771

**Comment by gitpower2017** [comment]
Hello, 
i have a problem with manual patching from  python-imaging to python-pil
Can i download new deb packet?
